test-services/check-links/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.16.0-alpine
```

Comment:
This would be more appropriate under the `scripts/docs` directory as it isn't really used by the integration tests.

Reply:
> This would be more appropriate under the `scripts/docs` directory as it isn't really used by the integration tests.

Will move to `scripts/docs` in a new PR.